.circleci/generate_circleci_config.py

```diff
@@ -15,6 +15,7 @@
     ("1.10.7-16.dev", ["alpine3.10", "buster"]),
     ("1.10.10-6.dev", ["alpine3.10", "buster"]),
     ("1.10.12-2.dev", ["alpine3.10", "buster"]),
+    ("1.10.13-1.dev", ["buster"]),
```

Comment:
We do not want to publish alpine image for 1.10.13, correct? @ashb @paolaperaza ?